src/controllers/house.controllers.js

```diff
@@ -12,4 +12,14 @@ const createNewHouseController = async (houseData) => {
   }
 };
 
+export const getAll = async () => {
+  const data = await House.find({}).exec();
```

Comment:
As we discussed, it'd probably best if it returned only the data needed in the house list and map.